.github/workflows/release_crates.yml

```diff
@@ -73,11 +73,13 @@ jobs:
           echo "::endgroup::"
 
           echo "::group::Tagging release using 'cargo release tag'"
-          cargo release tag --no-confirm --execute --sign-tag --tag-prefix ""
+          cargo release tag --no-confirm --execute --sign-tag --tag-prefix "" --exclude probe-rs-mi
+          cargo release tag --no-confirm --execute --sign-tag --package probe-rs-mi || true
```

Comment:
Why do we need the || true? :)

Reply:
`cargo release` fails with "no package selected" if probe-rs-mi has no changes. I'm attempting to not make the whole workflow fail here.

Reply:
Can we maybe add that as a comment for the next reader to not fall over their feet? :)